magda-sleuther-broken-link/src/sleuther.ts

```diff
@@ -0,0 +1,441 @@
+import Registry from "@magda/typescript-common/dist/Registry";
+import { Record } from "@magda/typescript-common/dist/generated/registry/api";
+import unionToThrowable from "@magda/typescript-common/dist/util/union-to-throwable";
```

Comment:
as in `import api from "@magda/typescript-common/dist/generated/registry/api"` and use it as `api.Record` ?